# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/util/APZCCallbackHelper.h
# Commit: 96da9d241051
# Full Hash: 96da9d241051d222bdd693bdb519d058a013b59c
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-23 09:46:24
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 8cefa694f811 -r 96da9d241051 gfx/layers/apz/util/APZCCallbackHelper.h
--- a/gfx/layers/apz/util/APZCCallbackHelper.h	Fri Mar 22 18:28:31 2019 +0000
+++ b/gfx/layers/apz/util/APZCCallbackHelper.h	Fri Mar 22 18:28:42 2019 +0000
@@ -37,7 +37,7 @@
  public:
   DisplayportSetListener(nsIWidget* aWidget, nsIPresShell* aPresShell,
                          const uint64_t& aInputBlockId,
-                         const nsTArray<ScrollableLayerGuid>& aTargets);
+                         const nsTArray<SLGuidAndRenderRoot>& aTargets);
   virtual ~DisplayportSetListener();
   bool Register();
   void DidRefresh() override;
@@ -46,7 +46,7 @@
   RefPtr<nsIWidget> mWidget;
   RefPtr<nsIPresShell> mPresShell;
   uint64_t mInputBlockId;
-  nsTArray<ScrollableLayerGuid> mTargets;
+  nsTArray<SLGuidAndRenderRoot> mTargets;
 };
 
 /* This class contains some helper methods that facilitate implementing the